### DISCUSSION TITLE
Provide a Homebrew package

### DISCUSSION THREAD

**Discussion by ajoual** [f1multiviewer/issue-tracker#98] [Feature requests]
In order to allow users a simple way to stay up to dat with new releases, a Homebrew formula could be provided for MacOS users and perhaps something similar could be done for Linux users too.

**Comment by jeppeolesen** [comment]
Closing this - the windows and mac builds now contain an auto-updater, and the team won't be maintaining separate update channels to keep the workload as light as possible.
